### ISSUE TITLE
Screen Rotation and UI

### ISSUE THREAD

**Issue by noobww** [plateaukao/einkbro#196]
Sreen rotation between portrait and landscape causes abnormal touch-to-pagedown/up area height.

**Comment by noobww** [comment]
![Uploading Screenshot_20221227-220329.png…]()

**Comment by Cwpute** [comment]
I have the same problem.
In my case it happens specifically when i push the fullscreen button on a video and turn my smartphone to its side for convenience. When i get out of fullscreen, a bar will have appeared on either side of the phone, permanently there umtel i start the app again.

Here is a first picture. I went fullscreen, turned the phone to its side, immediately turned it back to portrait, then exited the fullscreen: 
![Screenshot_20221230-230010](https://user-images.githubusercontent.com/80604512/210115020-d7804794-5a00-4183-83c1-af6ba62956e6.png)

Then i got back to fullscreen, turn the phone on its side, the exited fullscreen in landscape mode. I got gifted with a second bar !
![Screenshot_20221230-231048](https://user-images.githubusercontent.com/80604512/210115060-caf99d0a-e60b-4587-b6f9-6dc636dc2e30.png)

I'm on Android 11, using a Nokia TA-1157.
